src/utils/OpReturnUtils.ts

```diff
@@ -67,13 +67,16 @@ export function isValidOpReturn(
   for (let i = 0; outputs && i < outputs.length; i += 1) {
     const output: NormalizedOutput = outputs[i];
     if (output.op_return_data
-      && output.op_return_data.length === 92
+      && output.op_return_data.length >= 50
```

Comment:
Let's restrict it to the two valid length only
```suggestion
      && (output.op_return_data.length === 50 || output.op_return_data.length === 92)
```